### ISSUE TITLE
Save as...

### ISSUE THREAD

**Issue by p0m3e3** [adobe/brackets#14489]
Great program.
Make the save mode as, as well as the notepad++
[https://prnt.sc/k9fizz](url)
It will be very useful.
thanks in advance

**Comment by navch** [comment]
Hi @p0m3e3 Are you talking about save session? You can find extensions for this in brackets.

**Comment by p0m3e3** [comment]
@navch Yes, it's about this. Can you tell me which one is needed?

**Comment by navch** [comment]
I just read about notepad++ save session feature. I thought it's about autosaving a file in a session. But It actually keeps track of what files are opened in Notepad++, with some extra information like file positions and highlighter for the file. I guess there is no similar feature but you get the recent files opened with the shortcut (ctrl+tab). Which also maintains the selection of recent files. If you are looking to autosave a file then there is an extension called "Autosave every edit!", If you want to save all open files on app switch then you can install other extension called "Autosave Files on Window Blur". Hope this helps. Can you actually explain exactly what you want maybe we consider that as feature idea?
If you want to see the file which contains all these information of last opened file,  selection information then you can check %appdata%/Brackets/state.json

**Comment by p0m3e3** [comment]
@mavch No. You did not understand me. I know about Autosave. I need that, if selecting save as ..., there was a choice. For example. .js ?, .py, .html .css, and others

**Comment by navch** [comment]
Ohh Sorry. Got it. For now, this doesn't exist. But looks like this can be done easily via an extension as well. I will update you shortly on this.
